src/renderer/ssf-api.ts

```diff
@@ -236,6 +237,28 @@ export class SSFApi {
         }
     }
 
+    /**
+     * Allows JS to register a log retriever that can be used by electron main process.
+     * @param  {Object} retriever  function that can be called to retrieve logs
+     */
+    public registerLogRetriever(retriever): void {
```

Comment:
Put a type on the argument

Reply:
What type should it have? I did it the same way as the existing registerProtocolHandler which looks like this:
`public registerProtocolHandler(protocolHandler): void`